components/admins/modals/upgradeModal.tsx

```
@@ -47,13 +48,34 @@
+  height: Yup.number()
+    .typeError('Height must be a number')
+    .required('Height is required')
+    .integer('Must be a valid number')
+    .test(
+      'min-height',
+      'Height must be at least 1000 blocks above current height',
+      function (inputHeight) {
+        const proposedHeight = Number(inputHeight);
+        const chainHeight = Number(this.options.context?.chainData?.currentHeight || 0);
+
+        if (isNaN(proposedHeight) || isNaN(chainHeight)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)